### PR TITLE
Search result component

### PR DESCRIPTION
**Pull request by meganhng**

### CODE REVIEW

**Comment by sttic** [comment]
Putting this screenshot here for reference purposes:

image

File: package-lock.json [hunk 1]
```diff
@@ -1,8 +1,8921 @@
 {
   "name": "yokoso",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```
Comment (sttic) [inline package-lock.json:4]: This package-lock diff :eyes: Did you happen to update npm recently?

Apparently v2 for lockfile version should be backwards compatible so I think it should be ok for anyone using an older version :crossed_fingers: . We should keep this in mind if anyone starts getting package/install issues.

**Review by sttic — APPROVED** [review]
Cool! Nice small and large versions :rocket:

File: package-lock.json (hunk 1 above)
Comment (meganhng) [reply package-lock.json:4]: omg such a good catch! We'll keep a note of that, I think it is backwards compatible as you said so fingers crossed no issues occur. I'll be more careful next time

**Pull request merged**